utils/internal/download_gigaspeech_from_tsinghua.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+# Copyright 2021  Jiayu Du
+#                 Seasalt AI, Inc (Author: Guoguo Chen)
+
+set -e
+set -o pipefail
+
+stage=0
+with_dict=false
+
+. ./utils/parse_options.sh || exit 1
+
+if [ $# -ne 1 ]; then
+  echo "Usage: $0 <gigaspeech-dataset-dir>"
+  echo " e.g.: $0 /disk1/audio_data/gigaspeech"
+  echo ""
+  echo "This script downloads the entire GigaSpeech Dataset from Tsinghua host."
+  echo "We suggest having at least 1.2T of free space in local dir."
+  echo "If dataset resources are updated, you can just re-run this script for "
+  echo "incremental downloading, downloader will only download updates"
+  exit 1
+fi
+
+gigaspeech_dataset_dir=$1
+mkdir -p $gigaspeech_dataset_dir
+
+# Check operating system
+if [ `uname -s` != 'Linux' ]; then
+  echo "Tsinghua host supports *linux only*"
```

Comment:
Let's add "$0" when we print the error message, so that it will be easier to debug. E.g.,

echo "$0: The Tsinghua host only supports *linux* systems."